jekyll/_cci2/server/v4.2/installation/upgrade-server.adoc

```diff
@@ -55,6 +54,13 @@ kubectl -n <namespace> label secret/<secret-name> \
 
 . Check the link:https://circleci.com/server/changelog/[changelog] and make sure there are no actions you need to take before deploying a new version.
 
+. Export the "RabbitMQ" password as we are upgrading helm chart for the same as well.
```

Comment:
@atulsingh0 does this mean We are upgrading the helm chart for RabbitMQ? Not 100% sure what is meant by "the same" here that's all

Reply:
In this release, we are updating the RabbitMQ chart as well